package.json

```
@@ -4,7 +4,7 @@
   "description": "Put your photos on the map",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npm run build && DOTENV_CONFIG_PATH=./.test.env mocha -n require=dotenv/config",
```

I believe you don't need to build the code prior to running, just specify the compiler.
```suggestion
    "test": "DOTENV_CONFIG_PATH=./.test.env mocha -n -r dotenv/config -r ts-node/register",
```

---

Updated with ts-node in mocha config